extensions/ql-vscode/src/vscode-tests/no-workspace/cli.test.ts

```diff
@@ -0,0 +1,24 @@
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+
+import { CodeQLCliServer } from '../../cli';
+
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+
+describe.only('cliServerTests', () => {
+  it('should parse a valid SARIF file', async () => {
+    const result = await CodeQLCliServer.parseSarif(__dirname + '/data/sarif/validSarif.sarif');
+    expect(result.runs.length).to.eq(1);
```

Comment:
I think you want a different expectation here.

Reply:
I updated this. Hopefully it's a better test.